tests/l2scan_list/Makefile

```diff
@@ -0,0 +1,6 @@
+include ../Makefile.tests_common
```

Comment:
The test has to be moved to `tests/net` directory.

Reply:
Otherwise, the PR looks good to me. Are there any issues open? My test with a `nucleo-f411re` and the Adafruit WINC1500 Arduino shield worked as expected including GUA and LLUA.

Reply:
Once you have moved the test app to `tests/net`, please rebase and solve the conflict.

Reply:
Thanks for looking at it again.

> Are there any issues open?

Not as fat as I can tell. We also started to use the updated driver with still the static variant in production.
Nevertheless I found some tiny things worth to be fixed up.

One thing that I noted after a rebase to master is a pin conflict in my configuration on the `same54-xpro`.
The Ethernet IRQ pin (PD12) uses the same external interrupt controller (EXTINT[7]) as the EXT1 pin header interrupt pin (PB7) on the same54.
As a result no interrupt events had been generated. So if anyone uses this board, the EXT2 pin header should be used.
```
ATWINC15X0_PARAM_EXT2 = "{                                \
                        .spi         = SPI_DEV(1),        \
                        .spi_clk     = SPI_CLK_10MHZ,     \
                        .ssn_pin     = GPIO_PIN(2, 6),    \
                        .reset_pin   = GPIO_PIN(1, 1),    \
                        .irq_pin     = GPIO_PIN(3, 0),    \
                        .chip_en_pin = GPIO_PIN(1, 2),    \
                        .wake_pin    = GPIO_PIN(1, 6),    \
                      }" 
```